app/src/main/java/woowacourse/movie/view/home/AdvertisementViewHolder.kt

```diff
@@ -0,0 +1,17 @@
+package woowacourse.movie.view.home
+
+import android.view.View
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.recyclerview.widget.RecyclerView
+import woowacourse.movie.R
+
+class AdvertisementViewHolder(
+    view: View,
+    advertisementClickListener: () -> Unit,
+) : RecyclerView.ViewHolder(view) {
+    init {
+        view.findViewById<ConstraintLayout>(R.id.cl_advertisement).setOnClickListener {
+            advertisementClickListener.invoke()
+        }
```

Comment:
cl_advertisement가 root view라면 find하지 않고 view에 click listener를 달아도 됩니다.
```suggestion
        view.setOnClickListener {
            advertisementClickListener.invoke()
        }
```

Reply:
우와 좋은 피드백 감사합니다!!!

~~해당 내용 반영했습니다 : )~~

데이터 바인딩을 사용하며 현재는 제거됐습니다 ! ㅋㅋㅋ